tests/specs/integration/PinkUnicornsTests.cfc

```diff
@@ -18,32 +18,123 @@ component extends="tests.resources.BaseTest"{
+					} );
+				} );
+			} );
 
-				expect( event.getRenderedContent() ).toInclude( "SQL Logging</a>" );
-			});
+			story( "I need to get a list of cars", function(){
+				when( "I limit the result to 3", function(){
+					then( "I should get a list of cars", function(){
+						getRequestContext().setValue( "max", "3");
+						var e = get( "/slides/baseORMService" );
+						var prc = e.getPrivateCollection();
+						expect( prc.cars ).toBeArray().toHaveLength( 3 );
+					} );
+				} )
+			} );
 
+			story( "I need to create an active car", function(){
+				it( "Should be a component", function(){
+					var e 	= get( "/slides/activeEntity" );
+					var prc = e.getPrivateCollection();
+					expect( prc.newCar  ).toBeComponent();
+				} );
+			} );
 
-			it( "can leverage the base orm services", function(){
-				var event 	= get( "/slides/baseORMService" );
-				//var event 	= execute( route="/slides/baseORMService", renderResults=true );
-				//var event 	= execute( event="slides.baseORMService", renderResults=true );
-				var prc 	= event.getPrivateCollection();
+			story( "I need to find a specific active car", function(){
+				when( "I give a valid Car ID", function(){
+					then( "I should get a single active car", function(){
+						getRequestContext().setValue( "carID", "14");
+						var e = get( "/slides/activeEntity" );
+						var prc = e.getPrivateCollection();
+						expect( prc.myCar ).notToBeNull();
+					} );
+				} );
+			} );
 
-				expect( prc.newCar  ).toBeComponent();
-				expect( prc.newCar.getYear() ).toBe( 2012 );
+			story( "I need to get a list of active cars", function(){
+				when( "I limit the result to 3", function(){
+					then( "I should get a list of active cars", function(){
+						getRequestContext().setValue( "max", "3");
+						var e = get( "/slides/activeEntity" );
+						var prc = e.getPrivateCollection();
+						expect( prc.cars ).toBeArray().toHaveLength( 3 );
+					} );
+				} )
+			} );
 
-				expect( prc.myCar  ).notToBeNull();
+			story( "I need to create an virtual service", function(){
+				it( "Should be a component", function(){
+					var e 	= get( "/slides/virtualEntityService" );
+					var prc = e.getPrivateCollection();
+					expect( prc.newCar  ).toBeComponent();
+				} );
+			} );
 
-				expect( prc.cars )
-					.toBeArray()
-					.toHaveLength( 3 );
-			});
+			story( "I need to find a specific virtual service", function(){
+				when( "I give a valid Car ID", function(){
+					then( "I should get a single virtual service", function(){
+						getRequestContext().setValue( "carID", "14");
+						var e = get( "/slides/virtualEntityService" );
+						var prc = e.getPrivateCollection();
+						expect( prc.myCar ).notToBeNull();
+					} );
+				} );
+			} );
 
-		});
+			story( "I need to get a list of concrete Service", function(){
+				when( "I limit the result to 3", function(){
+					then( "I should get a list of concrete service", function(){
+						getRequestContext().setValue( "max", "3");
```

Comment:
Instead of passing this through here, pass it as the `params` argument to the `get()` operation.